src/routes/about.tsx

```diff
@@ -0,0 +1,109 @@
+import Page from '@/components/page'
+import { css } from 'linaria'
+
+const pageStyle = css`
+  text-align: center;
+  display: grid;
+  grid-template-columns: 1fr minmax(60%, 61rem) 1fr;
```

Comment:
@varCepheid can we use `margin: auto` combined with `width` and `min-width` to constrain its width and center it left-to-right instead of setting up a grid? Alternatively you could set up a grid with just one column and use `justify-content: center` to take up the extra space on the left and right without adding extra empty columns. Then we can avoid setting grid-column on the children.